### ISSUE TITLE
Add new way app delivery method: Brew?

### ISSUE THREAD

**Issue by saenkoyd** [kean/PulsePro#39]
We are implementing Pulse in our team, and it would be rather good to use something like Brew to install Pulse Pro on our team's Macs.

And for future updates, just use brew updating system instead of downloading from Github page

Thank you!

**Comment by kean** [comment]
Yes, I'm considering two options: putting it on App Store (unlikely to happen) and adding an auto-updater.

**Comment by saenkoyd** [comment]
Up to you 👍 

IMHO brew is more widespread and it's easier to install then App Store

**Comment by kean** [comment]
I didn't know that. I only ever user to install the command-line tools. I'll definitely give it a try then.

**Comment by techinpark** [comment]
I agree with the brew support, maybe I can contribute?

**Comment by lordcodes** [comment]
Brew is still really good for UI apps. It is worth mentioning this wouldn't just be a one-time set up though. It would likely involve either manual steps or some extra automation (and then checking it) for each new release.

Therefore, adding auto-updating into the app itself will also be very high value. That way the installation method just affects the first time install.
